.gitignore

```diff
@@ -38,6 +38,10 @@ node_modules
 # sftp configuration file
 sftp-config.json
 
+# OSX
+.DS_Store
+profile
```

Comment:
These should go into a global ignore for your machine, not in the repository: http://stackoverflow.com/a/7335487/99582